src/redux/books/booksSlice.js

```diff
@@ -1,44 +1,93 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { APP_ID, BASE_URL } from '../constants';
 
 const initialState = {
-  booksArr: [
-    {
-      item_id: 'item1',
-      title: 'The Great Gatsby',
-      author: 'John Smith',
-      category: 'Fiction',
-    },
-    {
-      item_id: 'item2',
-      title: 'Anna Karenina',
-      author: 'Leo Tolstoy',
-      category: 'Fiction',
-    },
-    {
-      item_id: 'item3',
-      title: 'The Selfish Gene',
-      author: 'Richard Dawkins',
-      category: 'Nonfiction',
-    },
-  ],
+  // booksList: [
+  //   {
+  //     item_id: 'item1',
+  //     title: 'The Great Gatsby',
+  //     author: 'John Smith',
+  //     category: 'Fiction',
+  //   },
+  //   {
+  //     item_id: 'item2',
+  //     title: 'Anna Karenina',
+  //     author: 'Leo Tolstoy',
+  //     category: 'Fiction',
+  //   },
+  //   {
+  //     item_id: 'item3',
+  //     title: 'The Selfish Gene',
+  //     author: 'Richard Dawkins',
+  //     category: 'Nonfiction',
+  //   },
+  // ],
```

Comment:
- [ ] [OPTIONAL] Kindly remove the commented code in order to keep your repo clean.